### ISSUE TITLE
Reaper shows all and every hidden track in mixer

### ISSUE THREAD

**Issue by smoochdaddy** [navelpluisje/reasonus-install#45]
Using the latest version of Reaper DAW and the latest version of Reasonus.  I have gotten my faderport 16 to sync up and work with Reaper on windows 10.  But all my tracks that were once hidden in the mixer window are now and always showing no how matter many times I hide the tracks not needed.  Everytime I sign into Reaper, the hidden tracks are visible in the mixer again in reaper.  Please help.  Thank you in advance.

Lou

**Comment by navelpluisje** [comment]
Hi @smoochdaddy Is the issue still there?

**Comment by smoochdaddy** [comment]
Yes, hi, it is still showing every hidden track.  I had to stop using
reasonus.  Is there anything I can do to get it to work.  I loved using
this feature you have. I can let you know anything you need to know.

Thank you,

Lou

**Comment by navelpluisje** [comment]
Have to see if i'm able to reproduce it. Will let you know. Otherwise there will be a new version of Reasonus in a couple of weeks. Faderport 8 is done, 16 almost and then have to do V2 and update the installer This is build upon CSI v3 and has no custom changes anymore in CSI.

**Comment by smoochdaddy** [comment]
Ok, awesome.  Please keep me posted and in the look.  Have a great rest of t
Your weekend.   I'm so excited, can't wait.

Enjoy,

Lou

**Comment by ClemMan10** [comment]
Hi! Just to let you know that the issue is still there and I started having the issue switching from Windows to Mac. I'm using FaderPort V2.
I don't know if it helps, anyway keep going you're awesome !
